vm/src/obj/objset.rs

```diff
@@ -813,7 +818,68 @@ impl TryFromObject for SetIterable {
     }
 }
 
+#[derive(Copy, Clone, Default)]
+struct SetSizeInfo {
+    size: Option<usize>,
```

Comment:
it seems this is intended to be None when this is invalidated (changed) but not used yet, right?

Reply:
Oh, I forgot to invalidate it... I just fixed it.